dev/manual_tests/lib/actions.dart

```diff
@@ -1,13 +1,19 @@
 // Copyright 2019 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+import 'dart:io';
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 
 void main() {
+  if (Platform.isMacOS) {
```

Comment:
Do these tests run on web? If so, we need the kIsWeb check to not break web, I believe.

Reply:
For the most part, these are "manual" tests, so they don't run unless we run them manually. There are a couple (`card_collection` and `color_testing_demo`) that are run as part of the smoke tests that I've now reverted so that they don't run on desktop (or web).